alf/experience_replayers/replay_buffer.py

```diff
@@ -203,16 +204,16 @@ def _change_mini_batch_length(self, mini_batch_length):
     # ``prioritized_sample()`` for the idx sampled to be still valid
```

Comment:
not ``idx`` anymore. It's possible to check whether positions are still valid by comparing with the current_pos.

Reply:
not idx anymore